src/storage/remote/protocol.h

```diff
@@ -29,24 +32,25 @@ Protocol commands for ProtocolServerHandler arrays passed to protocolServerProce
 #define PROTOCOL_COMMAND_STORAGE_INFO                               STRID5("s-i", 0x27730)
 #define PROTOCOL_COMMAND_STORAGE_LINK_CREATE                        STRID5("s-lc", 0x1b3730)
 #define PROTOCOL_COMMAND_STORAGE_LIST                               STRID5("s-l", 0x33730)
-#define PROTOCOL_COMMAND_STORAGE_OPEN_READ                          STRID5("s-or", 0x93f730)
-#define PROTOCOL_COMMAND_STORAGE_OPEN_WRITE                         STRID5("s-ow", 0xbbf730)
```

Comment:
Open is now a part of the command session rather than a separate command:
```
    {.command = PROTOCOL_COMMAND_STORAGE_WRITE, .open = storageRemoteWriteOpenProtocol,                                            \
        .processSession = storageRemoteWriteProtocol, .close = storageRemoteWriteCloseProtocol},
```
So instead of the PROTOCOL_COMMAND_STORAGE_OPEN_WRITE command we have the PROTOCOL_COMMAND_STORAGE_WRITE command with a session and the first thing we do is open that session.

Reply:
Right, so an implied `OPEN`; WFM...

Reply:
The open is still very much explicit. See https://github.com/dwsteele/pgbackrest/blob/dev-proto-multi/src/storage/remote/read.c#L216. It's just that it is no longer a separate command but just the session open for that command.